src/index.html

```diff
@@ -12,6 +12,120 @@
   </head>
   <body>
     <h1>HTML Form</h1>
+    <form class="form" >
+      <section class="section">
+        <div class="title">Personal informatiom:</div>
+        <div class="input">
+          <label>Surname
+            <input type="text" autocomplete="off" name="sureName">
```

Comment:
```suggestion
            <input 
              type="text" 
              autocomplete="off" 
              name="sureName"
            >
```
Each attribute should be placed on a new line if their amount is >= 3.
Fix all cases

Reply:
Доброго дня. Коли  я запускаю  npm test на компютері в мене не показує результат.
я його можу побачити тільки коли роблю push на GitHab тому порівняти не віходіть.Допоможіть вірішіті питання